app/controllers/admins/rooms_controller.rb

```diff
@@ -36,12 +38,9 @@ def update
   end
 
   def destroy
-    if @room.destroy
-      flash[:success] = t "admins.delete_success"
-    else
-      flash[:danger] = t "admins.delete_success"
-    end
-    redirect_to admins_rooms_path
+    @result = false
+    @result = true if @room.destroy
```

Comment:
```suggestion
        @result = @room.destroy ? true : false
```
không sử dụng @result có thể check @room đã được xóa ở bên view kh e

Reply:
Cái này em đc merge nó khác, ko hiểu sao rebase nó lỗi, tức quá anh ạ